src/identity/Identity.ts

```diff
@@ -51,21 +85,89 @@ export default class Identity extends PublicIdentity {
+   * [STATIC] Builds a new Identity, generated from a seed string as a hex.
    *
    * @param seedArg The seed as hex string. (Starting with 0x)
+   * @returns `Identity` built from the seed
+   * @example
+   *
+   * ```javascript
+   *
+   * // Using the Mnemonic below to generate an Identity to check if the seed is the same.
+   *
+   * const mnemonic = "coast ugly state lunch repeat step armed goose together pottery bind mention";
+   *
+   * const testerIdentity = Kilt.Identity.buildFromMnemonic(mnemonic);
+   *
+   * // testerIdentity.address 5HXfLqrqbKoKyi61YErwUrWEa1PWxikEojV7PCnLJgxrWd6W
+   * // testerIdentity.seedAsHex = 0x6ce9fd060c70165c0fc8da25810d249106d5df100aa980e0d9a11409d6b35261
+   *
+   * const testerIdentityCheck = Kilt.Identity.buildFromSeed(tester.seed);
+   *
+   * // testerIdentityCheck.address 5HXfLqrqbKoKyi61YErwUrWEa1PWxikEojV7PCnLJgxrWd6W
+   * // testerIdentityCheck.seedAsHex = 0x6ce9fd060c70165c0fc8da25810d249106d5df100aa980e0d9a11409d6b35261
+   * // The address is the same and the seed is the same.
+   * ```
    */
   public static buildFromSeedString(seedArg: string) {
     const asU8a = hexToU8a(seedArg)
     return Identity.buildFromSeed(asU8a)
   }
-
+  /**
+   * [STATIC] Builds a new Identity, generated from a seed.
+   * @param seed
+   * @returns `Identity`
+   * @example
+   *
+   * ```javascript
+   *
+   * // Using the Mnemonic below to generate an Identity to check if the seed is the same.
+   *
+   * const mnemonic = "coast ugly state lunch repeat step armed goose together pottery bind mention";
+   *
+   * const testerIdentity = Kilt.Identity.buildFromMnemonic(mnemonic);
+   *
+   * // testerIdentity.address 5HXfLqrqbKoKyi61YErwUrWEa1PWxikEojV7PCnLJgxrWd6W
+   * // testerIdentity.seed = seed: Uint8Array [
+   * // 108, 233, 253,  6,  12, 112,  22,  92,
+   * // 15, 200, 218, 37, 129,  13,  36, 145,
+   * // 6, 213, 223, 16,  10, 169, 128, 224,
+   * // 217, 161,  20,  9, 214, 179,  82,  97
+   * // ],
+   *
+   * const testerIdentityCheck = Kilt.Identity.buildFromSeed(tester.seed);
+   *
+   * // testerIdentityCheck.address 5HXfLqrqbKoKyi61YErwUrWEa1PWxikEojV7PCnLJgxrWd6W
+   * // testerIdentityCheck.seed = seed: Uint8Array [
+   * // 108, 233, 253,  6,  12, 112,  22,  92,
+   * // 15, 200, 218, 37, 129,  13,  36, 145,
+   * // 6, 213, 223, 16,  10, 169, 128, 224,
+   * // 217, 161,  20,  9, 214, 179,  82,  97
+   * // ],
+   * // The address is the same and the seed is the same.
+   * ```
+   */
   public static buildFromSeed(seed: Uint8Array) {
     const keyring = new Keyring({ type: 'ed25519' })
     const keyringPair = keyring.addFromSeed(seed)
     return new Identity(seed, keyringPair)
   }
-
+  /**
+   * [STATIC] Builds a new Identity, generated from a uniform resource identifier (URIs).
+   * @param uri A phrase built from a mnemonic string.
+   * @returns `Identity`
+   * @example
+   * ```javascript
+   * // An Identity of Bob
```

Comment:
Suggestion: the comment here is extraneous, I think

Reply:
Okie dokie